catalog/transformer/catalog-transformer-tika-input/src/main/java/ddf/catalog/transformer/input/tika/TikaInputTransformer.java

```diff
@@ -385,7 +385,7 @@ public Metacard transform(InputStream input, String id)
 
         processContentMetadataExtractors(bodyText, metacard);
 
-        processMetadataExtractors(bodyText, metacard);
+        processMetadataExtractors(metadataText, metacard);
```

Comment:
There is also a case where the metadataText could be empty string.
Maybe add a return on StringUtils.isBlank inside processMetadataExtractors.
Also add a similar check inside processContentMetadataExtractors.